_posts/2021-04-21-decoding-from-memory-bug.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+published: true
+title:  "Solidity ABI Decoder Bug For Multi-Dimensional Memory Arrays"
+date:   2021-04-21
+author: Solidity Team
+category: Security Alerts
+---
+
+https://www.certora.com/blog/deserialization.html
+
+On April 5th, 2021, a bug in the Solidity abi decoder v2 was reported by
+John Toman of the Certora development team. Certora's bug disclosure post
+can be found [here](https://www.certora.com/blog/deserialization.html).
+
+The bug is fixed with [Solidity version 0.8.4](https://github.com/ethereum/solidity/releases/tag/v0.8.4)
+released on April 21st, 2021. **The bug is present in all prior versions of abi coder v2.**
```

Comment:
It's `abi coder v2` here and `ABI coder V2` in another place. I think one form should be used consistently.

```suggestion
released on April 21st, 2021. **The bug is present in all prior versions of ABI coder V2.**
```